.github/workflows/wheels.yml

```diff
@@ -204,21 +202,11 @@ jobs:
           path: ./wheelhouse/*.whl
 
       - name: Upload wheels & sdist
-        if: ${{ success() && (env.IS_SCHEDULE_DISPATCH == 'true' || env.IS_PUSH == 'true') }}
-        shell: bash -el {0}
-        env:
-          PANDAS_STAGING_UPLOAD_TOKEN: ${{ secrets.PANDAS_STAGING_UPLOAD_TOKEN }}
-          PANDAS_NIGHTLY_UPLOAD_TOKEN: ${{ secrets.PANDAS_NIGHTLY_UPLOAD_TOKEN }}
-        # trigger an upload to
-        # https://anaconda.org/scientific-python-nightly-wheels/pandas
-        # for cron jobs or "Run workflow" (restricted to main branch).
-        # Tags will upload to
-        # https://anaconda.org/multibuild-wheels-staging/pandas
-        # The tokens were originally generated at anaconda.org
-        run: |
-          source ci/upload_wheels.sh
-          set_upload_vars
-          upload_wheels
+        if: ${{ success() && env.IS_SCHEDULE_DISPATCH == 'true' }}

```

Comment:
The removal of `|| IS_PUSH` here just means that we no longer will push a tagged commit to the nightly channel? (which I suppose is fine, although strictly speaking then there is a brief moment that the nightly channel is behind on PyPI 🤷 )

Reply:
> The removal of || IS_PUSH here just means that we no longer will push a tagged commit to the nightly channel?

I suppose so, yes. IMO the day lag between indexes should hopefully be OK. Not sure if other projects have a policy of also publishing their stable releases on the nightly index.